pkgs/applications/audio/roc-toolkit/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, scons
+, python
+, ragel
+, gengetopt
+, libuv
+, libpulseaudio
+, sox
+}:
+
+stdenv.mkDerivation {
+  name = "roc-toolkit";
+
+  nativeBuildInputs = [ scons python ragel gengetopt ];
+  buildInputs = [ libuv libpulseaudio sox ];
+
+  src = fetchFromGitHub {
+    owner = "roc-streaming";
+    repo = "roc-toolkit";
+    rev = "c89687330bfce6f4dce79826f7a235b581f2b49d";
+    sha256 = "1j2gqfpsh0hrwrwvanv6b893b9pds93rv7bl7am2050j3jqmfjrw";
+  };
+
+  prefixKey = "--prefix=";
```

Comment:
This is the default, you shouldn't to set this explicitly

Reply:
The default is missing the `--` and compilation fails.